SimPEG/Mesh/TensorMesh.py

```diff
@@ -217,7 +229,7 @@ def isInside(self, pts, locType='N'):
             inside = inside & (pts[:,i] >= tensor.min()-TOL) & (pts[:,i] <= tensor.max()+TOL)
         return inside
 
-    def getInterpolationMat(self, loc, locType='CC', zerosOutside=False):
+    def _getInterpolationMat(self, loc, locType='CC', zerosOutside=False):
```

Comment:
Could alternatively call `super` or `BaseTensorMesh.getInterpolationMat(self, ...)`  for future reference. I think this is fine.